build-system/compile/compile.js

```diff
@@ -328,8 +322,13 @@ function compile(entryModuleFilenames, outputDir, outputFilename, options) {
         'constantProperty',
         'globalThis'
       );
+      compilerOptions.jscomp_warning.push('accessControls');
```

Comment:
can we add a comment somewhere linking to the explanation of these error/warning groups.

Reply:
Good idea. Done.